### PR TITLE
hy/test SAP google adclick

### PR DESCRIPTION
**Pull request by sv-hyacoub**
# Description

This test ensures that adclick data is collected accordingly for Google.

## Bugzilla bug ID

**ID: 1902777**
**Link: https://bugzilla.mozilla.org/show_bug.cgi?id=1902777**

## Type of change

Please delete options that are not relevant.

- [x] New Test

# How does this resolve / make progress on that bug?

Completed.

# Screenshots / Explanations

N/A.

# Comments / Concerns

I'm using time.sleep before clicking on "Accept cookies" otherwise it can't find the element, I tried using element_exists or nav.expect(ec.title_contains("Google Search")) but that didn't help.

### CODE REVIEW

File: modules/data/navigation.components.json [hunk 1]
```diff
@@ -146,5 +146,17 @@
         "groups": [
             "doNotCache"
         ]
+    },
+
+    "accept-cookies": {
+    "selectorData": "L2AGLb",
```
Comment (ben-c-at-moz) [inline modules/data/navigation.components.json:152]: Generally, we don't want to select on ids or classes that look like this--often a base64 number or hash. These are generated by various web frameworks and can vary from version to version of the website, or even by the session. I suspect this is why it is failing in CI.

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]

File: modules/data/navigation.components.json (hunk 1 above)
Comment (emilghittasv) [reply modules/data/navigation.components.json:152]: Good point. Hmm I can't find a better strategy for that element other than it's visible text.

Can we change the strategy to xpath and the selectorData to "//div[text()='Accept all']" ? It should work.

Comment (sv-hyacoub) [reply modules/data/navigation.components.json]: I just made a commit with all the requested changes.

**Review by emilghittasv — DISMISSED** [review]
lgtm

**Review by emilghittasv — APPROVED** [review]

**Pull request merged**